src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = days * 40;
+  const firstBreakPoint = 3;
+  const secondBreakPoint = 7;
```

Comment:
Let's make more descriptive namings.
```suggestion
  const daysForSmallDiscout = 3;
  const daysForBigDiscout = 7;
```